kubetest2-kind/deployer/deployer.go

```diff
@@ -63,6 +63,17 @@ type deployer struct {
 	verbosity      int    // --verbosity for kind
 }
 
+func (d *deployer) Kubeconfig() (string, error) {
```

Comment:
this isn't quite correct vs kind / client-go FYI ... https://github.com/kubernetes-sigs/kind/tree/master/pkg/cluster/internal/kubeconfig/internal/kubeconfig

Reply:
are you referring to https://github.com/kubernetes-sigs/kind/blob/master/pkg/cluster/internal/kubeconfig/internal/kubeconfig/paths.go#L30-L43 ?

I think we are only missing the KUBECONFIG env handling from there. will add as a follow up. This should solve the basic case when nothing is set.

Reply:
~HOME is also more complex on windows, off the top of my head.

I think this deployer should probably have it's own stable computed paths and use those consistently (like $HOME/.kube/kubetest2-kind-$cluster)

Reply:
yeah, I'm planning on sending kind deployer specific changes separately soon and the top-level artifacts organization changes, will fix this more robustly there.

Current change is only to make quick testing of the clusterloader2 tester locally with the kind deployer easy.